### DISCUSSION TITLE
Lab Assignment #5

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/21S-ASIAAM-191A#143] [Assignments]
Paste your GitHub pages URL for Week_06 as a comment below to submit your [lab assignment #5](https://github.com/albertkun/21S-ASIAAM-191A/blob/main/Week_6/Lab/lab_assignment.md).

e.g. https://akochaphum.github.io/21S-ASIAAM-191A-Assignments/Week_06/index.html

**Comment by rachan2023** [comment]
https://rachan2023.github.io/21S-ASIAAM-191A-Assignments/Week_6/index.html

**Comment by KenNL42** [comment]
https://kennl42.github.io/21S-ASIAAM-191A-Assignments/Week_06/index.html

**Comment by ghost** [comment]
https://ktgreenler.github.io/21S-ASIAAM-191A-Assignments/Week_6/index.html

**Comment by jason-muljadi** [comment]
https://jason-muljadi.github.io/21S-ASIAAM-191A-Assignments/Week_06/index.html

**Reply by skarakas1** [reply]
This is awesome--the font/color scheme makes me feel like I'm on someone's family tree hobby website circa 1998!

**Comment by skarakas1** [comment]
https://skarakas1.github.io/21S-ASIAAM-191A-Assignments/Week_06/index.html

**Reply by alizezila** [reply]
wow, this looks awesome!

**Reply by skarakas1** [reply]
thank you!

**Comment by alizezila** [comment]
https://alizezila.github.io/21S-ASIAAM-191A-Assignments/Week_06/index.html

**Comment by rh5140** [comment]
https://rh5140.github.io/21S-ASIAAM-191A-Assignments/Week_06/index.html

(also... let me know if it looks okay on your screen resolution D: it's acceptable on one screen I checked on but not the other...)

EDIT (5/13/2020 12:50PM)
Thanks Albert! It has been fixed ;w;

**Reply by albertkun** [reply]
@rh5140 it looks good except i think the `grids` might be a little off, i would check the debugger:

![image](https://user-images.githubusercontent.com/8574425/118032014-09373000-b31c-11eb-91f2-d10b008e4080.png)

**Reply by rh5140** [reply]
![image](https://user-images.githubusercontent.com/40257341/118049218-49a1a880-b332-11eb-95ef-350882b718a7.png)
hmmm above is 16:9 (I think).. I changed the row size but it's still always wonky on what I think is 16:10...

**Reply by albertkun** [reply]
@rh5140 it might be related to this grid cell:
![image](https://user-images.githubusercontent.com/8574425/118064486-f5a5bc80-b34f-11eb-8ff1-33862d894fb3.png)

**Reply by albertkun** [reply]
@rh5140 maybe try removing the `grid-template-rows: 1fr 8fr 8fr;` line in the `body {}`? or change it to something like:

```css
body{
grid-template-rows: .1fr .8fr .1fr;
}
```

**Reply by rh5140** [reply]
@albertkun thanks, removing the rows fixed it!

**Comment by nnhimichelle** [comment]
https://nnhimichelle.github.io/21S-ASIAAM-191A-Assignments/Week_06/index.html

**Reply by albertkun** [reply]
@nnhimichelle hmm, the buttons dont work, because you have two `content` divs

**Reply by nnhimichelle** [reply]
oops! I think it should work now :'0

**Reply by albertkun** [reply]
@nnhimichelle yep!

**Comment by lvalles10** [comment]
https://lvalles10.github.io/21S-ASIAAM-191A-Assignments/Week_6/index.html

**Comment by epham36** [comment]
https://epham36.github.io/21S-ASIAAM-191A-Assignments/Week_6/index.html

**Comment by latishasumardy** [comment]
https://latishasumardy.github.io/21S-ASIAAM-191A-Assignments/Week_6/index.html

**Reply by albertkun** [reply]
@latishasumardy omg, sleep!

**Reply by latishasumardy** [reply]
@albertkun u should sleep too !! :))

**Reply by albertkun** [reply]
@latishasumardy touche.. just about to! you better be too!

**Comment by jsxu315** [comment]
https://jsxu315.github.io/21S-ASIAAM-191A-Assignments/Week_06/index.html

**Reply by albertkun** [reply]
@jsxu315 AHHH! sleeeep!

**Comment by okcirE** [comment]
https://okcire.github.io/21S-ASIAAM-191A-Assignments/Week_6/index.html

**Comment by Wny-Duong** [comment]
https://wny-duong.github.io/21S-ASIAAM-191A-Assignments/Week_6/index.html

**Comment by avaitt** [comment]
https://avaitt.github.io/21S-ASIAAM-191A-Assignments/Week_06/index.html

**Comment by ninalong** [comment]
https://ninalong.github.io/21S-ASIAAM-191A-Assignments/Week_6/index.html

**Comment by brillianz** [comment]
https://brillianz.github.io/21S-ASIAAM-191A-Assignments/Lab_04/index.html

**Reply by skarakas1** [reply]
Ritchie Valens! Live at Pacoima Jr High! Legendary!

**Comment by vcta99** [comment]
https://vcta99.github.io/21S-ASIAAM-191A-Assignments/Week_06/index.html

**Comment by annyrviloria** [comment]
https://annyrviloria.github.io/21S-ASIAAM-191A-Assignments/Week_06/index.html

Sorry I am a little late, my comps exam melted my brain :-(

**Reply by albertkun** [reply]
@annyrviloria no problem! great submission though!! 👍
